src/media/css/previews-tray.styl

```diff
@@ -1,7 +1,8 @@
 @import 'lib';
 
-$dot-size = 13px;
-$triangle-size = 20px;
+$bar-height = 3px;
+$primary-blue = #4cb0ff;
```

Comment:
This is defined as action-positive but I plan on bringing back the primary-X names as kind of a super base color over action-positive